javascript/javascript3/week3/CirclePaint/script.js

```diff
@@ -0,0 +1,58 @@
+// @ts-nocheck
+const canvas = document.getElementById('canvas');
+const context = canvas.getContext('2d');
+
+const windowHeight = window.innerHeight;
+const windowWidth = window.innerWidth;
+
+canvas.height = windowHeight;
+canvas.width = windowWidth;
+
+canvas.style.background = 'yellow';
+class Circle {
+  constructor(x, y, r, startAngle, endAngle) {
+    this.x = x;
+    this.y = y;
+    this.r = r;
+    this.startAngle = startAngle;
+    this.endAngle = endAngle;
+  }
+  draw() {
+    context.beginPath();
+    context.arc(this.x, this.y, this.r, this.startAngle, this.endAngle);
+    context.strokestyle = randomPosition();
+    context.fillStyle = randomColor();
+    context.fill();
+    context.closePath();
+  }
+}
+
+//Create a circle
+function randomColor() {
+  let r = (255 * Math.random()) | 0,
+    g = (255 * Math.random()) | 0,
+    b = (255 * Math.random()) | 0;
+  return 'rgb(' + r + ',' + g + ',' + b + ')';
+}
+randomColor();
+
+function randomPosition() {
+  random_x = Math.random() * windowHeight;
+  random_y = Math.random() * windowWidth;
+  random_r = Math.random() * 100;
```

Comment:
If you a little change code in lines 40 and 41, circles will be drawn across the entire width of the screen.

random_x = Math.random() * windowWidth;
random_y = Math.random() * windowHeight;

Reply:
Thank you so much Julia, it works fine now, its fill to the entire screen